src/tokenManagement/tokenManagementController.ts

```diff
@@ -0,0 +1,495 @@
+    }
+
+    parseLocalMint(transactionOnNetwork: TransactionOnNetwork): { tokenIdentifier: string }[] {
+        return this.parser.parseLocalMint(transactionOnNetwork);
+    }
+
+    async createTransactionForLocalBurning(
+        sender: IAccount,
+        nonce: bigint,
+        options: resources.LocalBurnInput,
+    ): Promise<Transaction> {
+        const transaction = this.factory.createTransactionForLocalBurning(sender.address, options);
+
+        transaction.nonce = nonce;
+        transaction.signature = await sender.sign(this.txComputer.computeBytesForSigning(transaction));
+
+        return transaction;
+    }
+
+    async awaitCompleteLocalBurn(txHash: string): Promise<{ tokenIdentifier: string }[]> {
+        const transaction = await this.transactionAwaiter.awaitCompleted(txHash);
+        return this.parseLocalBurn(transaction);
+    }
+
+    parseLocalBurn(transactionOnNetwork: TransactionOnNetwork): { tokenIdentifier: string }[] {
+        return this.parser.parseLocalBurn(transactionOnNetwork);
+    }
+
+    async createTransactionForUpdatingAttributes(
+        sender: IAccount,
+        nonce: bigint,
+        options: resources.UpdateAttributesInput,
+    ): Promise<Transaction> {
+        const transaction = this.factory.createTransactionForUpdatingAttributes(sender.address, options);
+
+        transaction.nonce = nonce;
+        transaction.signature = await sender.sign(this.txComputer.computeBytesForSigning(transaction));
+
+        return transaction;
+    }
+
+    async awaitCompletedUpdateAttributes(txHash: string): Promise<{ tokenIdentifier: string }[]> {
+        const transaction = await this.transactionAwaiter.awaitCompleted(txHash);
+        return this.parseUpdateAttributes(transaction);
+    }
+
+    parseUpdateAttributes(transactionOnNetwork: TransactionOnNetwork): { tokenIdentifier: string }[] {
+        return this.parser.parseUpdateAttributes(transactionOnNetwork);
+    }
+
+    async createTransactionForAddingQuantity(
+        sender: IAccount,
+        nonce: bigint,
+        options: resources.UpdateQuantityInput,
+    ): Promise<Transaction> {
+        const transaction = this.factory.createTransactionForAddingQuantity(sender.address, options);
+
+        transaction.nonce = nonce;
+        transaction.signature = await sender.sign(this.txComputer.computeBytesForSigning(transaction));
+
+        return transaction;
+    }
+
+    async awaitCompletedAddQuantity(txHash: string): Promise<{ tokenIdentifier: string }[]> {
+        const transaction = await this.transactionAwaiter.awaitCompleted(txHash);
+        return this.parseAddQuantity(transaction);
+    }
+
+    parseAddQuantity(transactionOnNetwork: TransactionOnNetwork): { tokenIdentifier: string }[] {
+        return this.parser.parseAddQuantity(transactionOnNetwork);
+    }
+
+    async createTransactionForBurningQuantity(
+        sender: IAccount,
+        nonce: bigint,
+        options: resources.UpdateQuantityInput,
```

Comment:
The input should be updated here, same as for the transactions factory.

Reply:
done